daffodil-cli/src/main/scala/org/apache/daffodil/InfosetTypes.scala

```diff
@@ -358,7 +356,7 @@ case class JDOMInfosetHandler(dataProcessor: DataProcessor)
 }
 
 class JDOMInfosetParseResult(parseResult: ParseResult, output: JDOMInfosetOutputter)
-  extends InfosetParseResult(parseResult) {
+    extends InfosetParseResult(parseResult) {
```

Comment:
This is interesting indenting extends 4 spaces instead of 2. Maybe that's a scala standard but I don't think we've been doing that. Not sure what the preference here.

Reply:
The default configuration tends to like 4 spaces on continuations like this. I changed a few of them from 4 to 2 but I may have missed a few.

Reply:
I'm not sure what the standard is, but I did find the parameter: `indent.extendSite`. I set it to 2 for now so you can see how that looks.